client/src/views/Profile/Card/index.vue

```diff
@@ -1,18 +1,31 @@
 <template>
   <div class="profile-card">
-      <div class="user-avatar">
-        <img :src="profile.avatar_url" alt="avatar"/>
-<!--        <i v-if="popularAccount" class="fas fa-star" title="Popular account"></i>-->
-      </div>
+    <div class="user-avatar">
+      <avatar :img="profile.avatarUrl || ''" :size="'140px'" />
```

Comment:
If size isn't dynamic - no need to bind it, could simply pass it as attribute aka size="140px"